documentation/Celledoni.tex

```diff
@@ -340,24 +352,232 @@ \subsubsection*{The elliptic modulus}
+the rotation diagrams above are modified as follows for $\mathscr Q_t$:
+\[
+\begin{tikzcd}
+{\mathscr Q_t: \mathscr B^b} \arrow{r}{\mathscr S} & {\mathscr B^p}\arrow{r}{\mathscr P_t} & {\mathscr B_t} \arrow{r}{\mathscr Y_t} & {\mathscr B'} \arrow{r}{\mathscr R} & {\mathscr B^s}
+\end{tikzcd}
+\]
+And for $\mathscr R$:
+\[
+\begin{tikzcd}
+{\mathscr R: \mathscr B'} \arrow{r}{\mathscr Y_{t_0}^{-1} = \Identity} & {\mathscr B_t} \arrow{r}{\mathscr P_{t_0}^{-1}} & {\mathscr B^p} 
+\arrow{r}{\mathscr S^{-1}} & {\mathscr B^b} \arrow{r}{\mathscr Q_{t_0}} & {\mathscr B^s}
+\end{tikzcd}
+\] 
+
+\subsection*{Integrals}
+We start by computing two integrals that are useful to obtain $\gy\of{t}$.  They are valid for $0 \le a < 1$.  As far as we can tell the following integral,
+which is useful for cases (i) and (ii),
+is missing from \cite{ByrdFriedman1954}:
+\begin{align}
+\int{}\frac{1}{1 + a \JacobiDN\of{u, k}}\diffd{u} &= \nonumber\\
+\frac{1}{1 - a^2}&
+\begin{aligned}[t]
+\;\Biggl[&\EllipticPi\of{\JacobiAmplitude\of{u, k}, \frac{a^2 k^2}{a^2 - 1}, k} - \\
+&a\sqrt\frac{1 - a^2}{a^2\pa{k^2 - 1} + 1} \InverseTrigonometricTangent\of{\sqrt\frac{a^2 \pa{k^2 - 1} + 1}{1 - a^2} \JacobiSC\of{u, k}}\Biggr]
+\end{aligned}
+\label{integraldn}
+\end{align}
+Also, the following integral is useful for case (iii):
+\begin{equation}
+\int{}\frac{1}{1 + a \HyperbolicSecant\of{u}}\diffd{u} = u + \frac{2 a}{\sqrt{1 - a^2}} 
+\InverseTrigonometricTangent\of{\frac{a - 1}{\sqrt{1 - a^2}} \HyperbolicTangent\of{\frac{u}{2}}}
+\label{integralsech}
+\end{equation}
+
+\subsection*{Case (i)}
+In case (i), we define $\mathscr P_t$ to map $\vm$ onto $\VectorSymbol{e_1}$.  A computation similar to that in \cite{Celledoni2007} yields that rotation
+in quaternionic form:
+\[
+\begin{dcases}
+p_2 &= \frac{p_1 m_2 + p_0 m_3}{G + m_1} \\
+p_3 &= \frac{p_1 m_3 - p_0 m_2}{G + m_1} \\
+p_0^2 + p_1^2 &= \frac{G + m_1}{2 G}
+\end{dcases}
+\]
+and, for the angle $\gy\of{t}$ of the rotation $\mathscr Y_t$ around $\VectorSymbol{e_1}$:
+\[
+\TimeDerivative{\gy}\of{t} = \frac{2 T + G m_1 / I_1}{G + m_1} + 4 G \frac{p_1 \TimeDerivative{p_0} - p_0 \TimeDerivative{p_1}}{G + m_1}
+\]
+We then write $p_0 = c_0 \sqrt{1 + m_1/G}$ and $p_1= c_1 \sqrt{1 + m_1/G}$ and pick $c_0 = 1/\sqrt{2}$ and $c_1 = 0$.  The quaternion simplifies to:
+\[
+\begin{dcases}
+p_0 &= \sqrt\frac{1 + m_1/G}{2} \\
+p_1 &= 0 \\
+p_2 &= \frac{m_3}{\sqrt{2 G\pa{G + m_1}}} \\
+p_3 &= \frac{-m_2}{\sqrt{2 G\pa{G + m_1}}}
+\end{dcases}
+\]
+and the angle to:
+\[
+\TimeDerivative{\gy}\of{t} = \frac{2 T + G m_1 / I_1}{G + m_1} = \frac{G^2 - \gD_1 + G m_1}{I_1\pa{G + m_1}} = \frac{G}{I_1} - \frac{\gD_1/I_1}{G + m_1} =
+\frac{G}{I_1} - \frac{\gD_1}{G I_1}\frac{1}{1 + m_1/G}
+\]
+Using equation (\ref{integraldn}) with $a = B_{13}/G$ and simplifying the various coefficients we obtain:
+\[
+\gy\of{t} = \frac{G}{I_1}t + \frac{G I_{13}}{\gl I_1 I_3}
+\EllipticPi\of{\JacobiAmplitude\of{\gl t - \gn}, \frac{I_1 I_{32}}{I_3 I_{12}}, k} -
```

Comment:
s/𝑘/𝑘<sup>-1</sup>/

Reply:
I don't think so, I think you meant around line 560.

Reply:
Also, I noticed that the second parameter of am(*u*, *k*) was missing in a few places.